README.md

```diff
@@ -0,0 +1,44 @@
+![Aptabase](https://aptabase.com/og.png)
+
+# Unity Engine SDK for Aptabase
+
+Instrument your Unity Engine project with Aptabase, an Open Source, Privacy-First and Simple Analytics for Mobile, Desktop and Web Apps.
+
+## Install
+
+Import the Unity Package from Releases.
```

Comment:
What do you mean by Releases? a GitHub release?

I'm not familiar with Unity packages, but it seems like NuGet is not a thing in Unity?

Reply:
Package manager seems like it's easier for users? Doesn't have to be done in this PR though, we can follow up on that separately.